client/js/Utils/sc_helper.js

```diff
@@ -13,24 +13,24 @@ ScHelper.prototype.getConnectorElements = async function (arc) {
       arc,
       [sc.ScType.Unknown, "_target"]
   );
-  const result = await scClient.templateSearch(scTemplate);
+  const result = await scClient.searchByTemplate(scTemplate);
   return [result[0].get("_source"), result[0].get("_target")];
 };
 
 /*! Check if there are specified arc between two objects
  * @param {String} addr1 sc-addr of source sc-element
- * @param {int} type type of sc-edge, that need to be checked for existing
+ * @param {int} type type of sc-connector, that need to be checked for existing
  * @param {String} addr2 sc-addr of target sc-element
- * @returns Function returns Promise object. If sc-edge exists, then it would be resolved; 
+ * @returns Function returns Promise object. If sc-connector exists, then it would be resolved; 
  * otherwise it would be rejected
  */
-ScHelper.prototype.checkEdge = async function (addr1, type, addr2) {
+ScHelper.prototype.checkConnector = async function (addr1, type, addr2) {
   let template = new sc.ScTemplate();
   addr1 = new sc.ScAddr(addr1);
   type = new sc.ScType(type).changeConst(false);
```

Comment:
probably this method works incorrect, maybe create an issue if you agree
it searches for template with triple that has variable connector and returns true if triple found(equal behavior for both const and variable connectors), but shouldn't it search for triple with connector that has not specified constancy and then check if any of result constructions has connector with type that was passed?

Reply:
Method changeConst(false) changes const subtype with var subtype. This method can be used if you want to search for constructions with constant sc-connectors only

Reply:
Ok